Makefile

```diff
@@ -60,7 +60,7 @@ generate-vendor: ## Update/generate the vendor by using the path $GOPATH/src/sig
 .PHONY: generate-setup
 generate-setup: ## Current workarround to generate the testdata with the correct controller-gen version
 	- rm -rf $(CONTROLLER_GEN_BIN_PATH)
-	- GO111MODULE=on go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.1
+	- GO111MODULE=on go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.2
```

Comment:
Should this be part of a different PR? Bumping controller-gen to v0.2.2 doesn't seem related to the import fix.

Reply:
Hi Adirio, the command `make generate-setup` is just a helper for development purpose. We can use it before exec the golden script to ensure that the correct version of the `controller-gen` will be used. So, it is a very small update and since it is too small I believe that would be ok be in this PR.